adapters/incoming/rest/src/test/java/be/xplore/pricescraper/controllers/ItemControllerTests.java

```diff
@@ -16,38 +18,30 @@
 import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.http.HttpStatusCode;
 
-@SpringBootTest
+@SpringBootTest(classes = ItemsController.class)
 @Slf4j
 public class ItemControllerTests {
   @Autowired
   private ItemsController itemsController;
   @MockBean
-  private ItemPriceRepository itemPriceRepository;
+  private ItemService itemService;
```

Comment:
We have decided to mock the service here so we could provide custom responses from the api, so no network requests are performed. It might have been better to have this in the service tests themselves instead of the controller however